core/controllers/acl_decorators_test.py

```diff
@@ -61,6 +61,12 @@ class PlayExplorationDecoratorTests(test_utils.GenericTestBase):
 
     class MockHandler(base.BaseHandler):
         GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+        URL_PATH_ARGS_SCHEMAS = {
+            'exploration_id': {
+                'type': 'unicode'
```

Comment:
Yes, URL args are always going to be a unicode, but since our project's requirements is to validate all types of data coming to a handler class, i.e., either from payload or request or URL path arguments.
Thus following this requirement, I have written a schema for URL path arguments, looking forward to your point.